backend/default.nix

```diff
@@ -0,0 +1,44 @@
+{ haskell-nix, runCommand, lib, hlint }:
+let
+  ignoreFilter = path: type:
```

Comment:
Do you understand why we need this `ignoreFilter`?

Reply:
Cargo culted from something I wrote in 2017. It's not necessary, and I'm removing it.